.github/workflows/openwiki-update.yml

```
@@ -0,0 +1,50 @@
+          ANTHROPIC_BASE_URL: ${{ secrets.OPENWIKI_ANTHROPIC_BASE_URL }}
+          LANGSMITH_API_KEY: ${{ secrets.OPENWIKI_LANGSMITH_API_KEY }}
+          LANGCHAIN_PROJECT: openwiki
+          LANGCHAIN_TRACING_V2: "true"
+
+      - name: Create OpenWiki update pull request
+        uses: peter-evans/create-pull-request@v7
+        with:
+          add-paths: |
+            openwiki
+          branch: openwiki/update
+          title: docs: update OpenWiki
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)